content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -57,14 +57,14 @@ The upgrade workflow at high level is the following:
 - You need to have a kubeadm Kubernetes cluster running version 1.16.0 or later.
 - [Swap must be disabled](https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux).
 - The cluster should use a static control plane and etcd pods or external etcd.
-- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md) carefully.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md) carefully.
```

Comment:
Should this CHANGELOG file version match the current docs release version (1.17) and use the proposed shortcode?
Ping @tengqm

Reply:
Yes, I changed that as well :+1: